arch/src/x86_64/cpu_profile.rs

```diff
@@ -18,7 +18,9 @@ use crate::x86_64::cpuid_definitions::{Parameters, deserialize_from_hex, seriali
 pub enum CpuProfile {
     #[default]
     Host,
+    #[cfg(feature = "kvm")]
```

Comment:
is it the right thing to feature gate them for KVM?

Reply:
Please reach out to Thomas and ask him about this. We do not have to take action in this PR but I think Thomas knows more background here and we can then persist this in some form of ticket